7.2/Dockerfile

```diff
@@ -228,7 +228,9 @@ RUN set -xe; \
 	apt-get clean; rm -rf /var/lib/apt/lists/*
 
 # PHP tools (installed globally)
-ENV COMPOSER_VERSION=1.10.7 \
+ENV COMPOSER_VERSION=2 \
```

Comment:
With backward compatibility in mind, we should not change the names of the existing variables or change their purpose. Let's stick with the following:

- `COMPOSER_DEFAULT_VERSION` - 1 or 2
- `COMPOSER_VERSION` - Composer v1 version (leave as is)
- `COMPOSER2_VERSION` - Composer v2 version